Essentials/src/messages_en.properties

```diff
@@ -601,6 +601,7 @@ whoisAFKSince=§6 - AFK\:§r {0} (Since {1})
 whoisBanned=§6 - Banned\:§r {0}
 whoisExp=§6 - Exp\:§r {0} (Level {1})
 whoisFly=§6 - Fly mode\:§r {0} ({1})
+whoisSpeed=§6 - Speed\:§r {0}
```

Comment:
Add messages to `messages.properties` only. Other language files should be edited on Crowdin (including `messages_en` and `messages_en_GB`).

Reply:
Fixed